src/locales/en-US/commands.json

```
@@ -34,53 +34,53 @@
     "hereIsYourDog": "Here's your dog!"
   },
   "invite": {
-    "commandDescription": "Shows the link to invite Switchblade to your server",
+    "commandDescription": "Shows the invite link for Switchblade.",
     "clickHere": "Click here to invite me to your server",
     "noteThat": "Note that you need the **$t(permissions:MANAGE_GUILD)** permission in order to invite bots."
   },
   "ping": {
-    "commandDescription": "Checks if the bot is online"
+    "commandDescription": "Checks if the bot is online."
   },
   "github": {
-    "commandDescription": "Searches for a user, repository or organization on Github",
+    "commandDescription": "Searches for a user, repository or an organization on Github.",
```

```suggestion
    "commandDescription": "Searches for a user, repository or a organization on Github.",
```
Is that last `an` really needed?

---

yes, because `o` is a vowel